lit/setup-and-operations/teams.lit

```diff
@@ -92,6 +92,18 @@ against the teams that have granted them access.
     This can be useful if you've forgotten your team name.
   }
 
+  \section{
```

Comment:
Suggest changing from `\section` to just a `bold`

Reply:
OK - I can make that change. I will roll it up today.